cookbooks/cdo-varnish/templates/default/default.vcl.erb

```diff
@@ -497,9 +503,9 @@ end
     } else if (req.url ~ "^/auth") {
       set req.http.Cookie = regsuball(req.http.Cookie, ";(cdo-private|hour_of_code|NO_CACHE)=", "; \1=");
     } else if (req.http.host == "<%= canonical_hostname('learn.code.org') %>" || req.http.host == "<%= canonical_hostname('studio.code.org') %>") {
-      set req.http.Cookie = regsuball(req.http.Cookie, ";(language_|<%= session_cookie %>|<%= storage_cookie %>|hour_of_code|NO_CACHE)=", "; \1=");
+      set req.http.Cookie = regsuball(req.http.Cookie, ";(language_|<%= session_cookie %>|<%= deprecated_storage_cookie %>|<%= storage_id_cookie %>|hour_of_code|NO_CACHE)=", "; \1=");
```

Comment:
I would favor clarity over brevity, but am open to specific suggestions if you want to shorten it without making it less clear